### PR TITLE
chore: Added a few lines on keeping in sync with the repo

### PR DESCRIPTION
**Pull request by Biki-das**
Signed-off-by: Biki-das <bikid475@gmail.com>

Many new contributors especially beginners who start to contribute usually face a problem in keeping the local repo to be in sync with the main repo , a small explaining line might help to them to keep their development workflow seamless.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,14 +66,22 @@ Click the icon at the right end of that box to copy that URL.
...
-it will display the link to your clone.
-   - Use the **Fetch upstream** dropdown menu
-to determine if your fork is current.
-   - If your fork is not current,
-use the **Fetch and merge** button to sync it.
+   Go to github and copy the url of the main keptn-github-io repo
+   ```   
+   https://github.com/keptn/keptn.github.io.git
+   ```
+   make sure to be in the rootfolder of the project and the branch should be master branch and type
```
Comment (StackScribe) [inline CONTRIBUTING.md:73]:
```suggestion
   Be sure that you are in the root folder of the project and in the master branch.
   Issue the following command:
```

**Review by StackScribe — COMMENTED** [review]
This needs review by a git expert.  Is setting upstream really an alternative to the Fetch and Merge functionality to keep ones fork in sync with master? 

We also need to sync these instructions with what is in keptn/keptn/CONTRIBUTING.md .  What do we want to recommend as a "default" procedure for people to use?

**Comment by Biki-das** [comment]
> This needs review by a git expert. Is setting upstream really an alternative to the Fetch and Merge functionality to keep ones fork in sync with master?
> 
> We also need to sync these instructions with what is in keptn/keptn/CONTRIBUTING.md . What do we want to recommend as a "default" procedure for people to use?

sure.

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Thanks! It looks right

**Comment by afzal442** [comment]
I am not Git expert though. I had few suggestion to add.
`git push origin master` doesn't look trivial here
I would suggest following guide https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md#make-a-pull-request

**Comment by Biki-das** [comment]
> I am not Git expert though. I had few suggestion to add.
> `git push origin master` doesn't look trivial here
> I would suggest following guide https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md#make-a-pull-request 
> cc @oleg-nenashev 

I have personally used this method from a long and never faced any issue, to keep the repo in sync!

**Comment by afzal442** [comment]
Good, but how one could sync feature branch with the base repo in case of resolving `conflict` in files? I think that cmd wouldn't alone solve that kind of case, you need to add `git rebase master` or `git merge master`.

**Pull request merged**